docs/NewAppointment.puml

```diff
@@ -0,0 +1,29 @@
+@startuml
+'https://plantuml.com/sequence-diagram
+
```

Comment:
I found many rounded rectangles instead of rectangles, from my knowledge, we have to use rectangles.